src/uuid/rfc4122.cr

```diff
@@ -0,0 +1,85 @@
+# Support for RFC 4122 UUID variant.
+struct UUID
+  # RFC 4122 UUID variant versions.
+  enum Version
```

Comment:
@jhass what do you mean?

I'm not sure an enum is needed is the versions are just numbers.